tests/test_position_sequence.py

```diff
@@ -0,0 +1,147 @@
+import pytest
+
+from useq import GridRelative, MDASequence
+
+
+@pytest.fixture
+def mda() -> MDASequence:
+    return MDASequence(
+        axis_order="tpgcz",
+        stage_positions=[(10, 20, 0), {"name": "test", "x": 10, "y": 20, "z": 50}],
+        channels=[{"config": "Cy5", "exposure": 50}],
+    )
+
+
+def _update_pos_sequence(mda: MDASequence, axis: str):
+    p_seq = MDASequence()
+    if "c" in axis:
+        p_seq = p_seq.replace(channels=[{"config": "DAPI", "exposure": 100}])
+    if "z" in axis:
+        p_seq = p_seq.replace(z_plan={"top": 60, "bottom": 55, "step": 1})
+    if "g" in axis:
+        p_seq = p_seq.replace(grid_plan={"rows": 2, "columns": 2})
+    if "t" in axis:
+        p_seq = p_seq.replace(time_plan=[{"interval": 1, "loops": 2}])
+    new_pos = [
+        (10, 20, 0),
+        {"name": "test", "x": 10, "y": 20, "z": 50, "sequence": p_seq},
+    ]
+    return mda.replace(stage_positions=new_pos)
+
+
+def event_list(sequence: MDASequence) -> list:
+    return [
+        (
+            i.global_index,
+            i.index,
+            i.pos_name,
+            i.x_pos,
+            i.y_pos,
+            i.z_pos,
+            i.min_start_time,
+            i.exposure,
+            # i.channel,
+        )
+        for i in sequence.iter_events()
+    ]
+
+
+def test_position_sequence_channels(mda: MDASequence) -> None:
+    mda1 = mda.replace(
```

Comment:
see above.  if you're just immediately going to override the fixture, don't use it.  make a new one here